r/src/main/java/org/apache/zeppelin/rinterpreter/KnitR.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.rinterpreter;
+
+import org.apache.zeppelin.interpreter.*;
+import org.apache.zeppelin.scheduler.Scheduler;
+
+import java.net.URL;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Created by aelberg on 7/28/15.
```

Comment:
Thank you! 
Good point -  though Moon's name is no different from others, it was there for historical reasons before joining ASF, but now committers are removing it gradually while touching such files. (Does not make sense to spend time and create a separate PR just for that reason)